scripts/linegraph.py

```diff
@@ -0,0 +1,28 @@
+import csv
+import matplotlib.pyplot as plt
+import pandas as pd
+
+plt.figure()
+
+file_names = ['sample/log-rps-1000-iv-10.csv','sample/log-rps-100-iv-10.csv','sample/log-rps-10-iv-10.csv']
```

Comment:
take file names as a list of command line arguments.  This way a user can run it with `python linegraph.py sample/log1.csv sample/log2.csv sample/log3.csv` or even just use `python linegraph.py sample/log*.csv` which your terminal will automatically expand into a list of file names that match that pattern separated by spaces.